modules/accessing-microshift.adoc

```diff
@@ -0,0 +1,46 @@
+= Accessing the MicroShift cluster
+== Installing the OpenShift client
+
+You can install the OpenShift CLI (oc) client on your RHEL machine by running:
+
+[source,terminal]
+----
+sudo dnf install -y openshift-clients
+----
+
+== Accessing the MicroShift cluster locally
+
+To access the MicroShift cluster from the _same_ machine that is running MicroShift, copy the generated `kubeconfig` file to your user's `~/.kube/config` file:
+
+[source,terminal]
+----
+mkdir -p ~/.kube/
+sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
+----
+
+== Accessing the MicroShift cluster remotely
+
+To access the MicroShift cluster from a _different_ machine from the one running MicroShift, use the following procedure:
+
+1. Open the Kubernetes API port in the firewall of your MicroShift machine:
+
+    MICROSHIFT_MACHINE=<name or IP of your MicroShift machine>
+    ssh $MICROSHIF_MACHINE "sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload"
+
+2. Copy the generated `kubeconfig` file from the MicroShift machine to your local machine:
+
+    mkdir -p ~/.kube/
+    scp $MICROSHIFT_MACHINE:/var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
```

Comment:
Will a non-root user be able to read the file this way? We needed `sudo` to `cat` the file earlier.

Reply:
Yes, it would not work. To simplify the instrucions, I would suggest logging into the MicroShift machine to perform firewall config and copy of kubebconfig to the user directory.
Then, you can copy it elsewhere from the host.